content/en/tracing/other_telemetry/connect_logs_and_traces/python.md

```diff
@@ -29,16 +29,29 @@ further_reading:
 
 ### Standard library logging
 
-To correlate your [traces][1] with your logs, update your log format to include
-the required attributes from the log record and call `ddtrace.patch(logging=True)`.
+To correlate your [traces][1] with your logs, follow the steps below:
+
+Step 1) Activate automatic instrumentation through any of the options below:
```

Comment:
Do we prefer users to generally use one of these three options? That could be good to call out to not overwhelm them with these choices. Or we could at least mention why you'd use one option over the others.

Reply:
I didn't want to get into which option to use because then we have to revisit the discussion for any documentation for any APM features/configurations like writing custom span tags, enabling DBM/APM, etc. The main doc https://docs.datadoghq.com/tracing/trace_collection/?tab=containers covers a few approaches already as you scroll through it.

There are actually more variations to set up the Python tracer, but I listed the three most common ones.